test/org/sagebionetworks/bridge/play/controllers/UserProfileControllerTest.java

```diff
@@ -39,28 +54,69 @@ public void canSubmitExternalIdentifier() throws Exception {
         assertEquals("application/json", result.contentType());
         assertEquals("{\"message\":\"External identifier added to user profile.\"}", contentAsString(result));
         
-        verify(optionsService).setOption(TEST_STUDY, "healthCode", 
-            ParticipantOption.EXTERNAL_IDENTIFIER, "ABC-123-XYZ");
+        verify(optionsService).setString(TEST_STUDY, "healthCode", EXTERNAL_IDENTIFIER, "ABC-123-XYZ");
     }
-    
+
+    @SuppressWarnings({ "rawtypes", "unchecked" })
     @Test
-    public void externalIdentifierVerifiesIdentifierExists() throws Exception {
-        Http.Context.current.set(TestUtils.mockPlayContextWithJson("{\"identifier\":\"\"}"));
```

Comment:
externalIdentifierVerifiesIdentifierExists() was deleted. Do we no longer need this test?